app/views/spendings/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<div>
+  <div>
+  <% @header = "Transactions" %>
+	<%= render "shared/header", locals: { header: @header }, back: true %>
+  <div class="mb-10">
+  <% if @spendings.empty?%>
+    <p class="text-center pt-16 text-xl">You don't have any transactions yet !!</p>
+  <% else %>
+    <ul class="flex flex-col justify-center pt-12 ">
+      <li class="w-[90%] mx-auto flex flex-col items-center gap-3 shadow-lg px-3 py-5  my-5 bg-white">
+          <div class="flex  justify-center items-center">
+              <div class="text-4xl me-4">
+                <%= image_tag(@group.icon, alt: "Group Icon", class: "img-fluid", style: "height: 100px; width:100px;") %>
+              </div>
```

Comment:
- [ ] For the transaction page. Kindly stick to the color, structure and pattern which was provided in the behance design. The aim of this is to see how well you can replicate a design.

- The top bar should have a blue background. The color used is `#3778c2`
- The "Add new transaction" button should be at the bottom of the page and stretched horizontally as in the design. The color used is `#5fb523`
- Kindly fix the structure of each transaction to be placed in a flex horizontally 

### Expected design:
![image](https://github.com/arnoldnekemiah/Budget-app/assets/50990925/c8e1da3e-f85f-4140-9a69-96485a3ca5b2)

### Y
![image](https://github.com/arnoldnekemiah/Budget-app/assets/50990925/b6cb5719-bd25-4cf4-acab-f01d571f7e4a)
our design:

Reply:
- [ ] After adding spending, the page changes and does not look like the given wireframe any longer, kindly fix this
![Screenshot from 2023-11-25 14-34-31](https://github.com/arnoldnekemiah/Budget-app/assets/66097371/39526db0-3846-41cc-a89e-4f8cfb717210)